projects/packages/woocommerce-analytics/src/class-woo-analytics-trait.php

```diff
@@ -433,11 +433,11 @@ public function process_event_properties( $event_name, $properties = array(), $p
 			)
 		);
 
-		$js = "{'_en': '" . esc_js( $event_name ) . "'";
-
-		// ch param is needed to identify ClickHouse queries in the JS Analytics library
+		// ch param is needed to identify ClickHouse queries in the JS Analytics library. Remove prefix also for ClickHouse
```

Comment:
Maybe we could emphasise that ClickHouse does not include the prefix so we should remove it.